widget/widgetAssets/js/iocExtractionConfigService.service.js

```diff
@@ -66,12 +67,12 @@
       }
     }
 
-    function getGblVarToKeyStoreMapping(widgetBasePath) {
+    function getGblVarToKeyStoreMapping(widgetBasePath){
```

Comment:
getGblVarToKeyStoreMapping updated function name getKeyStoreMetaData also update the json name as it is not clear and add comment what is the purpose

Reply:
DONE